src/modules/dashboard/components/CoinMachine/BuyTokens/BuyTokens.tsx

```diff
@@ -85,6 +104,10 @@ const validationSchema = (userBalance: number) =>
 const BuyTokens = ({ colony: { colonyAddress }, disabled }: Props) => {
   const { username, ethereal, walletAddress } = useLoggedInUser();
 
+  /* To add proper states later */
+  const isSale = false;
```

Comment:
Actually i think we already can find out (with Raul changes) if there is a sale ongoing.

Reply:
I had a look and also checked with Raul and it's not worth it for now. There is only partial logic there so no point in trying to finish it up now.